scripts/make_countries.js

```diff
@@ -149,13 +150,25 @@ for (int res = 0; res < MAX_RES + 1; res++) {
 
   BENCHMARK(polygonToCells_AllCountries1, 5, {
     for (int index = 0; index < ${polygons.length}; index++) {
-      H3_EXPORT(polygonToCells)(&COUNTRIES[index], res, 0, hexagons);
+      H3_EXPORT(polygonToCells)(&COUNTRIES[index], res, CENTER_CONTAINMENT, hexagons);
     }
   });
 
   BENCHMARK(polygonToCells_AllCountries2, 5, {
     for (int index = 0; index < ${polygons.length}; index++) {
-      H3_EXPORT(polygonToCellsExperimental)(&COUNTRIES[index], res, 0, hexagons);
+      H3_EXPORT(polygonToCellsExperimental)(&COUNTRIES[index], res, CENTER_CONTAINMENT, hexagons);
+    }
+  });
+
+  BENCHMARK(polygonToCells_AllCountries3, 5, {
+    for (int index = 0; index < ${polygons.length}; index++) {
+      H3_EXPORT(polygonToCellsExperimental)(&COUNTRIES[index], res, FULL_CONTAINMENT, hexagons);
+    }
+  });
+
+  BENCHMARK(polygonToCells_AllCountries4, 5, {
+    for (int index = 0; index < ${polygons.length}; index++) {
+      H3_EXPORT(polygonToCellsExperimental)(&COUNTRIES[index], res, OVERLAPPING, hexagons);
```

Comment:
The new modes are slower than center containment, unsurprisingly, but still manage to be faster than the old algo in some cases:
```
Res 0	-- polygonToCells_AllCountries1: 198222.000000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries2: 42474.800000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries3: 247580.600000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries4: 282227.200000 microseconds per iteration (5 iterations)
Res 1	-- polygonToCells_AllCountries1: 324889.800000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries2: 18640.000000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries3: 72821.200000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries4: 89494.400000 microseconds per iteration (5 iterations)
Res 2	-- polygonToCells_AllCountries1: 257009.400000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries2: 66020.600000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries3: 143866.400000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries4: 181394.600000 microseconds per iteration (5 iterations)
Res 3	-- polygonToCells_AllCountries1: 424776.400000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries2: 280514.200000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries3: 442108.400000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries4: 545078.600000 microseconds per iteration (5 iterations)
Res 4	-- polygonToCells_AllCountries1: 2035545.200000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries2: 1462475.200000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries3: 1969331.600000 microseconds per iteration (5 iterations)
	-- polygonToCells_AllCountries4: 2559085.000000 microseconds per iteration (5 iterations)
```